### PR TITLE
Adds unit test for items that can be equipped to suit storage slots that are missing sprites

### PR DESCRIPTION
**Pull request by ShizCalev**
## About The Pull Request

Adds a unit test find every item that can be equipped to a suit slot which is displaying a missing sprite / CS:S texture.

The test is presently fully functional. That said, the test is currently set to a warning / logging only mode and will not cause CI failure. 

There are currently 257 sprites found which need to be resolved. Once this number is whittled closer to zero, the test will be switched to live mode/CI failure.

## Why It's Good For The Game
Missing sprites are bad.

## Changelog

:cl: ShizCalev
code: Added a unit test to catch items that can be equipped to suit slots have missing textures.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/changeling/powers/transform.dm [hunk 1]
```diff
@@ -96,7 +96,6 @@
 /obj/item/changeling
 	name = "flesh"
 	slot_flags = ALL
-	allowed = list(/obj/item/changeling)
```
Comment (ShizCalev) [inline code/modules/antagonists/changeling/powers/transform.dm:99]: This didn't do anything, it is correctly set on `/obj/item/clothing/suit/changeling`

File: code/modules/unit_tests/_unit_tests.dm [hunk 2]
```diff
@@ -183,6 +183,7 @@
 #include "traitor.dm"
 #include "unit_test.dm"
 #include "wizard_loadout.dm"
+#include "suit_storage_icons.dm"
```
Comment (Mothblocks) [inline code/modules/unit_tests/_unit_tests.dm:186]: ABC order

File: code/modules/unit_tests/suit_storage_icons.dm [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+	var/list/already_warned_icons = list()
+	var/count = 1 //to be removed once the test goes live / into CI failure mode.
+	for(var/obj/item/item_path as anything in typecacheof(wearable_item_paths))
+		if(initial(item_path.item_flags) & ABSTRACT)
+			continue
+
+		var/worn_icon = initial(item_path.worn_icon) //override icon file. where our sprite is contained if set. (ie modularity stuff)
+		var/worn_icon_state = initial(item_path.worn_icon_state) //overrides icon_state.
+		var/icon_state = !isnull(worn_icon_state) ? worn_icon_state : initial(item_path.icon_state) //icon_state. what sprite name we are looking for.
```
Comment (Mothblocks) [inline code/modules/unit_tests/suit_storage_icons.dm:34]:
```suggestion
		var/icon_state = worn_icon_state || initial(item_path.icon_state) //icon_state. what sprite name we are looking for.
```

**Review by Mothblocks — APPROVED** [review]
Small nits, looks good aside from these

**Pull request merged**